cfme/web_ui/__init__.py

```diff
@@ -3639,9 +3639,11 @@ class StatusBox(object):
 
     """
     def __init__(self, name):
-        self.name = name
+        assert type(name) == str
```

Comment:
We suppose that user passes a string, therefore I would recommend replacing this construct with:

```python
self.name = str(name).lower()
```